compiler/optimizer/StructuralAnalysis.cpp

```diff
@@ -51,7 +51,7 @@ void TR_RegionAnalysis::simpleIterator (TR_Stack<int32_t>& workStack,
                                         bool doThisCheck)
    {
    SparseBitVector::Cursor cursor(vector);
-   for (cursor.SetToLastOne(); cursor.Valid(); cursor.SetToPreviousOne())
+   for (cursor.SetToFirstOne(); cursor.Valid(); cursor.SetToNextOne())
```

Comment:
Looking at the code there may be a small benefit to backwards iteration, but the main reason for the change is that the reverse traversal is only currently supported by the SparseBitVector::Cursor and not the BitVector::Cursor - it seems to be the only asymmetry in the API...